benevolent/ocr.py

```diff
@@ -21,4 +23,34 @@ def get_image_text(image: Image.Image) -> str:
     return pytesseract.image_to_string(image)
 
 
+def img_to_mono__norm_and_otsu(
```

Comment:
Personally I would just name this `img_to_mono_with_otsu` or just `img_to_mono` ? Since the rest of the stuff just happens internally within the function.

Reply:
i did get really excited about 大津 when i first found out about it is probably why i wanted the longer name.  we're not going to have any clashes, though, i don't think.